src/Agent/NewRelic/Agent/Core/Utilization/VendorInfo.cs

```diff
@@ -25,6 +25,7 @@ public class VendorInfo
         private const string ValidateMetadataRegex = @"^[a-zA-Z0-9-_. /]*$";
 #if NETSTANDARD2_0
         private const string ContainerIdV1Regex = @".*:cpu:/docker/([0-9a-f]{64}).*";
+        private const string ContainerIdV1FallbackRegex = @"([0-9a-f]{64})"; // This is the old regex that just looks for any 64-char alphanumeric string
```

Comment:
Minor nit in the comment: It's actually looking for a 64-char hexadecimal string... ;-)